misc/acrn-config/xmls/board-xmls/tgl-rvp.xml

```diff
@@ -0,0 +1,282 @@
+	#define PM1A_EVT_ACCESS_SIZE    0x2U
+	#define PM1B_EVT_SPACE_ID       SPACE_SYSTEM_IO
+	#define PM1B_EVT_BIT_WIDTH      0x0U
+	#define PM1B_EVT_BIT_OFFSET     0x0U
+	#define PM1B_EVT_ADDRESS        0x0UL
+	#define PM1B_EVT_ACCESS_SIZE    0x2U
+	#define PM1A_CNT_SPACE_ID       SPACE_SYSTEM_IO
+	#define PM1A_CNT_BIT_WIDTH      0x10U
+	#define PM1A_CNT_BIT_OFFSET     0x0U
+	#define PM1A_CNT_ADDRESS        0x1804UL
+	#define PM1A_CNT_ACCESS_SIZE    0x2U
+	#define PM1B_CNT_SPACE_ID       SPACE_SYSTEM_IO
+	#define PM1B_CNT_BIT_WIDTH      0x0U
+	#define PM1B_CNT_BIT_OFFSET     0x0U
+	#define PM1B_CNT_ADDRESS        0x0UL
+	#define PM1B_CNT_ACCESS_SIZE    0x2U
+	</PM_INFO>
+
+	<S3_INFO>
+	#define S3_PKG_VAL_PM1A         0x5U
+	#define S3_PKG_VAL_PM1B         0U
+	#define S3_PKG_RESERVED         0x0U
+	</S3_INFO>
+
+	<S5_INFO>
+	#define S5_PKG_VAL_PM1A         0x7U
+	#define S5_PKG_VAL_PM1B         0U
+	#define S5_PKG_RESERVED         0x0U
+	</S5_INFO>
+
+	<DRHD_INFO>
+	#define DRHD_COUNT              6U
+
+	#define DRHD0_DEV_CNT           0x1U
+	#define DRHD0_SEGMENT           0x0U
+	#define DRHD0_FLAGS             0x0U
+	#define DRHD0_REG_BASE          0xFED90000UL
+	#define DRHD0_IGNORE            true
+	#define DRHD0_DEVSCOPE0_TYPE    0x1U
+	#define DRHD0_DEVSCOPE0_ID      0x0U
+	#define DRHD0_DEVSCOPE0_BUS     0x0U
+	#define DRHD0_DEVSCOPE0_PATH    0x10U
+
+	#define DRHD1_DEV_CNT           0x1U
+	#define DRHD1_SEGMENT           0x0U
+	#define DRHD1_FLAGS             0x0U
+	#define DRHD1_REG_BASE          0xFED84000UL
+	#define DRHD1_IGNORE            false
+	#define DRHD1_DEVSCOPE0_TYPE    0x2U
+	#define DRHD1_DEVSCOPE0_ID      0x0U
+	#define DRHD1_DEVSCOPE0_BUS     0x0U
+	#define DRHD1_DEVSCOPE0_PATH    0x38U
+
+	#define DRHD2_DEV_CNT           0x1U
+	#define DRHD2_SEGMENT           0x0U
+	#define DRHD2_FLAGS             0x0U
+	#define DRHD2_REG_BASE          0xFED85000UL
+	#define DRHD2_IGNORE            false
+	#define DRHD2_DEVSCOPE0_TYPE    0x2U
+	#define DRHD2_DEVSCOPE0_ID      0x0U
+	#define DRHD2_DEVSCOPE0_BUS     0x0U
+	#define DRHD2_DEVSCOPE0_PATH    0x39U
+
+	#define DRHD3_DEV_CNT           0x1U
+	#define DRHD3_SEGMENT           0x0U
+	#define DRHD3_FLAGS             0x0U
+	#define DRHD3_REG_BASE          0xFED86000UL
+	#define DRHD3_IGNORE            false
+	#define DRHD3_DEVSCOPE0_TYPE    0x2U
+	#define DRHD3_DEVSCOPE0_ID      0x0U
+	#define DRHD3_DEVSCOPE0_BUS     0x0U
+	#define DRHD3_DEVSCOPE0_PATH    0x3aU
+
+	#define DRHD4_DEV_CNT           0x1U
+	#define DRHD4_SEGMENT           0x0U
+	#define DRHD4_FLAGS             0x0U
+	#define DRHD4_REG_BASE          0xFED87000UL
+	#define DRHD4_IGNORE            false
+	#define DRHD4_DEVSCOPE0_TYPE    0x2U
+	#define DRHD4_DEVSCOPE0_ID      0x0U
+	#define DRHD4_DEVSCOPE0_BUS     0x0U
+	#define DRHD4_DEVSCOPE0_PATH    0x3bU
+
+	#define DRHD5_DEV_CNT           0x0U
+	#define DRHD5_SEGMENT           0x0U
+	#define DRHD5_FLAGS             0x1U
+	#define DRHD5_REG_BASE          0xFED91000UL
```

Comment:
Since other DRHD will define 	"#define DRHDx_IGNORE            false", should also be defined for DRHD5?

Reply:
@jsun26intel  can you answer this concern?

Reply:
@binbinwu1 The DRHDx_IGNORE value is depends on the PCI BDF under device scope filed, and 
 Device scope belong to Hardware Unit Definition of DRHD. 
In this case, there is no pci device under device scope field , because DEV_SCOPE is 0 in DMAR table. So the DRHDx_IGNROE should not defined while DEV_SCOPE is 0.

Reply:
From struct initialization point of view, the .ignore member need a initial value. Although without this variable assignment the member could get false automatically because the whole struct was set to 0 initially, but this is not friendly for code review also coding style. We should have a default value for this.